docs/zh-CN/docs/sql-manual/sql-functions/table-functions/hdfs.md

```diff
@@ -0,0 +1,79 @@
+---
+{
+    "title": "hdfs",
+    "language": "zh-CN"
+}
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# `HDFS`
```

Comment:
参考如下格式重新组织文档（以下标题都为“必须”项，并且标题使用英文）
```
<version since="1.2.0">

## SHOW-ALTER

### Name

### Description

### Example

### Keywords

### Best Practice

</version>
```

Reply:
done